### ISSUE TITLE
ARP fails when multicast limit is 0.

### ISSUE THREAD

**Issue by nolanl** [zerotier/ZeroTierOne#683]
Broadcasts are enabled.

IPv6 ND does work, presumably due to special-casing.

The only other slightly odd config is that one host has bridging enabled, though zt0 on that host has a IP (set by ZT), and is not yet part of a bridge.

All nodes are Debian w/ 1.2.4.

**Comment by adamierymenko** [comment]
Hmm... Multicast-free operation was normally intended to be used with our special IPv6 addressing modes that embed the ZT address into the IPv6 address and emulate NDP. These dispense with the need for multicast entirely. We could exempt ARP, but the idea of multicast-free operation is to completely shut off all that code.

Thoughts?

**Comment by adamierymenko** [comment]
Hrm... the thing is broadcast is a special case of multicast... kind of. It uses the same code/infrastructure and is of course subject to multicast limit.

Let me take a look and see how hard this would be.

**Comment by nolanl** [comment]
I mean, I have an easy enough workaround, I just set a multicast limit.

I didn't realize broadcast is subject to the multicast limit.  Does that mean ARP breaks randomly when the number of activeBridge=True endpoints is greater than the multicast limit?

**Comment by adamierymenko** [comment]
No, bridges are treated separately and are always able to receive multicast/broadcast. Setting something as a bridge causes it to get a bit more traffic than usual for this reason.
